python/mirascope/llm/clients/anthropic/_utils.py

```diff
@@ -145,33 +145,39 @@ def _convert_tool_to_tool_param(tool: ToolSchema) -> anthropic_types.ToolParam:
     )
 
 
-PARAMS_TO_KWARGS: _base_utils.ParamsToKwargs = {
-    "temperature": "temperature",
-    "max_tokens": "max_tokens",
-    "top_p": "top_p",
-    "top_k": "top_k",
-    "seed": None,
-    "stop_sequences": "stop_sequences",
-    "thinking": None,
-}
-
-
 def prepare_anthropic_request(
+    *,
     model_id: AnthropicModelId,
     messages: Sequence[Message],
-    tools: Sequence[ToolSchema] | BaseToolkit | None = None,
-    format: type[FormattableT] | Format[FormattableT] | None = None,
-    params: Params | None = None,
+    tools: Sequence[ToolSchema] | BaseToolkit | None,
+    format: type[FormattableT] | Format[FormattableT] | None,
+    params: Params,
```

Comment:
Removed all the default None settings because this is an internal utility that should always be called with all arguments, there's no ergonomic benefit to defaults here. Required calling by argument name for all args for safety and consistency with other prepare_ methods. (Here and elsewhere)